### PR TITLE
Redid opcodes related to player input and cutscenes

### PR DESCRIPTION
**Pull request by husho**
Removed wrong functionality from 0373, added 03ef.

### CODE REVIEW

File: rwengine/src/ai/PlayerController.hpp [hunk 1]
```diff
@@ -55,6 +55,9 @@ class PlayerController : public CharacterController {
     // @todo not implemented yet
     bool isBusted() const;
 
+    void prepareForCutscene();
```
Comment (ShFil119) [inline rwengine/src/ai/PlayerController.hpp]: Can you add short comments what these function do?

Comment (husho) [reply rwengine/src/ai/PlayerController.hpp]: Will do

**Comment by ShFil119** [comment]
Needs to be rebased.

**Comment by husho** [comment]
Rebased

**Pull request merged**